code/lib/telemetry/src/package-json.ts

```diff
@@ -27,9 +27,3 @@ export const getActualPackageJson = async (packageName: string) => {
   const packageJson = await fs.readJson(resolvedPackageJson);
   return packageJson;
 };
-
-// Note that this probably doesn't work in Yarn PNP mode because @storybook/telemetry doesn't depend on @storybook/cli
-export const getStorybookCoreVersion = async () => {
-  const { version } = await getActualPackageVersion('@storybook/cli');
-  return version;
-};
```

Comment:
@shilman this is a function in telemetry but it was only used by `upgrade` which doesn't use it anymore, so I felt it was okay to remove it. Let me know if not.